controllers/topics.controller.js

```diff
@@ -0,0 +1,7 @@
+const { getTopicsData } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+  getTopicsData().then((data) => {
+    res.status(200).send(data);
```

Comment:
You need to send back your data in an object, with a key to give it some context. 

a suitable key for this would be `topics`